bin/upload-dist-prod.sh

```diff
@@ -29,7 +32,7 @@ function upload_staging() {
 }
 
 # build and upload missing staging packages
-for package in $(bin/list-all-packages.sh)
+for package in $(list_all | awk '{print $1}')
```

Comment:
maybe make this a function in list-all-packages? Since a lot of the callers seem to need it